docs/release/trg-0/kit-artifacts.md

```diff
@@ -0,0 +1,255 @@
+
+### Use Case / Domain explanation
+
+Mandatory for every KIT that is related to a use case and all graduation stages
+
+This gives more insights into the use case or domain itself.
+
+- Todays challenge
+- Values for taking the challenge
+- Benefits for OEM, SME, Solution Provider
+
+Deliverable: Status quo, challenge and the benefits from different perspectives
+
+`Example from BPDM:`
+Business Partner KIT comes from the use case Business Partner Data Management.
+
+Status Quo / Todays challenge:  
+Most of today’s automotive companies have to invest a significant amount to keep their individual partner master data updated and correct, since outdated or incorrect data records result in federal fines and wrong claims / billing processes. The industry spends a vast amount into an area that is non differentiating, regulated and executed redundantly across the partners.
+
+For this, cutting individual company costs by offering the golden master data record as a shared service and proactively reduce Business Partner risk via collaborative value-add services.
+
+This will become the industry’s largest golden record collection as a shared and cost-optimized service (quality tested by the biggest players).
+
+OEMs and small-medium enterprises can:
+
+- Reduce today’s individual investments in master data mgt.
+- Re-allocate freed-up resources to differentiating / critical tasks
+- Time to value within 6 months, due to standard interfaces and mapping tools (e.g. SAP)
+- Increase data quality and use industry collective wisdom to act pro-actively (e.g. fraud alerts)
+
+Solution Provider:
+
+- Reduce investments to integrate due to one Data Model and open interface.
+- The KIT enables the offering of value-add services and innovation based on golden record and collective wisdom.
+- Potential to scale customer group and access new market potentials via marketplace and shared service network.
+
+### Tutorials
+
+Mandatory for the 'Graduated' stage. Optional for 'sandbox' and 'incubating'
+
+A tutorial provides educational resources that provides step-by-step guidance on how to use a KIT. Tutorials can be delivered in various formats, such as videos or written text. It may also provide additional resources for further exploration of the topic.
+
+Deliverables:
+
+- Description
+- Tutorial video or screenshot documentation
+
+### Whitepaper
+
+Optional
+
+A whitepaper outlines the overall objectives of a KIT regarding a specific business problem. It provides readers with background information on the topic, identifies key challenges and solutions to those challenges, evaluates alternative approaches to solving the business problem at hand, and offers recommendations for action.
+
+Deliverable:
+
+- Description
+- Link to the whitepaper
+
+### Semantic Models
```

Comment:
Semantic Models are wrong in the business view. They are located on the technical layer and not on the business one.

Reply:
We've been communicating this for months, and we don't want to confuse or annoy the teams with topics that aren't very relevant - everyone has a heavy load of topics to manage for the next release. So, we made small changes to the structure by adding the architecture view. But Malte, we have this on our agenda, and we'll do it step by step. Additionally, we have several graphics and documents to change. You may remember our always-communicated KIT structure image.

Reply:
This is an essential point since Traceability uses a possible "new" structure. If this is a TRG, the Traceability Kit wouldn't follow this "old" structure and couldn't be released since it doesn't fulfill this TRG.